web/settings/settings.php

```diff
@@ -830,7 +830,7 @@
 							<h1>Sofort Laden</h1>
 						</div>
 					</div>
-					<div class="row" style="background-color:#febebe">
+					<div class="row" style="background-color:#febebe; color:#000000;">
```

Comment:
Statt die Änderung in jeder Zeile hinzuzufügen, wäre es nicht beser gewesen, das in eine CSS-Definition im `<head>` zu legen?

Reply:
Ja, da die Einträge aber keine spezielle CSS-Klasse besitzen, können sie nicht auf diesem Weg formatiert werden. Die Lösung ist auch nur "quick and dirty" umgesetzt, da die Einstellungen derzeit überarbeitet werden und die Seite so wie sie ist verschwindet.